### ISSUE TITLE
Html template mixes javascript and html

### ISSUE THREAD

**Issue by christian-rli** [rl-institut/WAM_APP_FRED#86]
Separate the two. Put the javascript in a separate file and just import it.

**Comment by christian-rli** [comment]
@jh-RLI , I see that you are currently working on the html file. Could you address this issue? Or if you're done working there, I can try to do it. We should address this before working even more on the javascript. @Bachibouzouk took care of some nice formatting. It would be possible to check for that automatically, once it's separate from the html. Also, it's just more tidy and clean in general.

**Comment by jh-RLI** [comment]
Okay, I just changed the german text in Power Plant tab and added a description to time series. I was about to implement a [load spinner](https://github.com/makinacorpus/Leaflet.Spin/) but that would change the javascript. I will wait and continue in local testing mode :) @christian-rli

**Comment by christian-rli** [comment]
Thank you @jh-RLI . We decided against separating html and javascript after all. This is a nice to have for later on. I ran into several problems when trying to do this. 

1. {%%}-Syntax won't work in static (javascript) files. So using variables from python would be a pain. This could be resolved by only defining variables in the html and extracting all the remaining javascript elsewhere.

2.   I extracted a few functions into an external js-file and they appeared to work, albeit with funny/unpredictable shortcomings. E.g. weather points would load, but only with a very long delay, whereas the same code in one file would produce those points very quickly. Also points would be displayed, but hovering over them would no longer change their style. 

Overall it would be too bothersome to find and fix all of these little errors now and we should make do with a suboptimal language mix in our code.

¯\_(ツ)_/¯

closing this issue for now. Html file is free to be edited again.
